Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
+
+    bin.install "_bin/bitrise"
+  end
+
+  test do
+    output = shell_output(bin/"bitrise --version")
```

Comment:
One last question @bfontaine : we're working on the 0.9.8 version of `bitrise` which will automatically do a `setup` in case the user forgot to call `setup` before calling `bitrise run`, because our beta testers forgot this frequently. Should we still go with building these dependencies, like the way you described?

With `0.9.8` the `test do` should work without any further setup, because the setup check is included in the "run" action.